app/controllers/admin/action_pages_controller.rb

```diff
@@ -18,6 +18,11 @@ class Admin::ActionPagesController < Admin::ApplicationController
   allow_collaborators_to :index, :edit
 
   def index
+    @categories = Category.all.order(:title).map { |c| [c.title, c.id] }
+    @authors = User.joins(:action_pages).order(:last_name).map do |a|
+      label = a.name.blank? ? a.email : a.name
+      [label, a.id]
+    end
```

Comment:
Formatting view helper arguments in the controller like this is a bit unconventional. It'd be better for User to have an `.authors` scope and `#display_name` method, and feed these into Rails' `options_from_collection_for_select` helper.